internal/app/cf-terraforming/cmd/generate.go

```diff
@@ -701,6 +701,34 @@ func generateResources() func(cmd *cobra.Command, args []string) {
 								}
 
 								jsonStructData[i].(map[string]interface{})["rules"].([]interface{})[ruleCounter].(map[string]interface{})["action_parameters"].(map[string]interface{})["overrides"].(map[string]interface{})["enabled"] = nil
+
+								if jsonStructData[i].(map[string]interface{})["rules"].([]interface{})[ruleCounter].(map[string]interface{})["action_parameters"].(map[string]interface{})["overrides"].(map[string]interface{})["categories"] != nil {
```

Comment:
we have these nested `if`s above; are we not able to extend them instead of looping again?